lib/puppet_webhook.rb

```diff
@@ -87,13 +87,7 @@ class PuppetWebhook < Sinatra::Base
     data = JSON.parse(decoded, quirks_mode: true)
```

Comment:
This might be for a future improvement but I'd expect that if there's a WebhookJsonParser that it'd do all of it.

Reply:
I do believe that `Rack::Parser` is already doing the work. We just need to update our code to add what we need to the params `Rack::Parser` builds